### DISCUSSION TITLE
How Can I participate in the Challange

### DISCUSSION THREAD

**Discussion by Blockchain-Oracle** [Cyfrin/foundry-full-course-cu#1344] [Q&A] [answered]
I cant find the challange link how do  I participate.. The Qr code only takes me to etherScan can someone explain the process ?

**Comment by ghost** [comment, marked as answer]
In the block explorer you need to check the contract tab and then you need to read the contracts to find out how to solve the challenge. For now only solve the ones on Sepolia.

Challenges addresses https://github.com/Cyfrin/foundry-full-course-f23/discussions/135
